src/Debug.js

```diff
@@ -63,3 +73,61 @@ export function debugNode(node, indentLength = 2) {
 export function debugNodes(nodes) {
   return nodes.map(debugNode).join('\n\n\n');
 }
+
+export function debugInst(inst, indentLength = 2) {
+  if (typeof inst === 'string' || typeof inst === 'number') return escape(inst);
+  if (!inst) return '';
```

Comment:
hmm.... not that I'm aware of.

I don't actually know what calling this on `<div>{true}</div>` would do, but it should do whatever react itself does...

No idea how or why or what symbols would be doing getting called in this function....?